src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteController.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.companieshouse.web.accounts.controller.smallfull;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.web.accounts.controller.BaseController;
+import uk.gov.companieshouse.web.accounts.service.transaction.TransactionService;
+import uk.gov.companieshouse.web.accounts.util.RequestPathResolver;
+
+@Controller
+@RequestMapping(value = "/company/{companyNumber}/small-full/steps-to-complete")
+public class StepsToCompleteController {
+
+    private static final String TEMPLATE = "smallfull/stepsToComplete";
+
+    @Autowired
+    TransactionService transactionService;
```

Comment:
Can you set the private modifier on the private variables please

Reply:
Done